lib/experimental/Navigation/Carousel/index.tsx

```diff
@@ -60,6 +61,10 @@ export const Carousel = ({
     return showPeek ? (`peek${value || 1}` as PeekVariant) : value || 1
   }
 
+  if (!columns) {
+    return <DynamicCarousel>{children}</DynamicCarousel>
```

Comment:
Shouldn't we be passing the same props of the carousel here? So things like `onMouseEnter` still work and all that. Probably `opts` as well, and everything else.

Reply:
mmm.. Carousel don't have that props.. is just to make the plugin of autoplay works... I can try to implement and autoplay to our new component, but not in that PR, right?